scripts/changelog.js

```diff
@@ -75,7 +75,6 @@ function main () {
     if (/^---$/.test(line)) {
       print_commit(commit)
     } else if (m = line.match(/^([a-f0-9]{7,9}) ([a-f0-9]+) (?:[(]([^)]+)[)] )?(.*?) [(](.*?)[)]/)) {
-
```

Comment:
Someone else sent a fix for the same issue but it hasn't been merged. - https://github.com/npm/npm/pull/16110